osu.Game.Rulesets.Mania/Skinning/Argon/ArgonJudgementPiece.cs

```diff
@@ -98,7 +98,7 @@ public virtual void PlayAnimation()
             ringExplosion?.PlayAnimation();
         }
 
-        public Drawable? GetAboveHitObjectsProxiedContent() => null;
+        public Drawable? GetAboveHitObjectsProxiedContent() => JudgementText;
```

Comment:
As per test failures, this is wrong (due to not creating a proxy), but more importantly, why is this even required? As far as I can see, mania's judgements are in an entirely different container than the columns and objects, and they should already be in front of the hitobjects even without proxying. In fact `LegacyManiaJudgementPiece` returns `null` for this method. Am I missing something?

Keep in mind that appearances may be misleading (because the judgement text is additive in argon, it may *look* like it's behind the objects, but I'm pretty sure it's not).

Reply:
I was mostly just changing it for conformity, as you suggest it is not required.